contracts/src/tests/test_unruggable_memecoin.cairo

```diff
@@ -501,13 +501,15 @@ mod erc20_entrypoints {
 mod memecoin_entrypoints {
```

Comment:
Can you split this file here ?
create test_memecoin_erc20.cairo
and remove  mods erc20_metadata erc20_entrypoints from this file.

Reply:
so that we have a dedicated test file for specific memecoins functions (and avoid a 1500 lines test file)

Reply:
yeah i'll do this when I do the memecoin tests